book/01-introduction/sections/help.asc

```diff
@@ -1,46 +1,46 @@
 
-If you ever need help while using Git, there are two equivalent ways to get the comprehensive manual page (manpage) help for any of the Git commands:
+اگر در هنگام استفاده از گیت همیشه به کمک نیاز دارید، برای دستیابی به راهنمای دستی جامع (صفحه من)هر یک از دستورات گیت دو روش معادل وجود دارد :
 
-[source,console]
+[منبع، کنسول]
 ----
 $ git help <verb>
 $ man git-<verb>
 ----
 
-For example, you can get the manpage help for the `git config` command by running(((git commands, help)))
+به عنوان مثال، برای اجرای دستور پیکربندی گیت، شما می توانید به راهنمای دستی جامع با اجرا کردن آن دسترسی پیدا کنید.(((گیت دستورات, کمک)))
 
-[source,console]
+[منبع، کنسول]
 ----
-$ git help config
+$ گیت پیکربندی کمک
 ----
 
-These commands are nice because you can access them anywhere, even offline.
-If the manpages and this book aren't enough and you need in-person help, you can try the `#git` or `#github` channel on the Freenode IRC server (irc.freenode.net).
-These channels are regularly filled with hundreds of people who are all very knowledgeable about Git and are often willing to help.(((IRC)))
+.این دستورات بسیار خوب هستند، زیرا شما می توانید در هر جایی، حتی در حالت آفلاین به آنها دسترسی پیدا کنید
+اگر صفحه manpages و این کتاب کافی نباشند و شما نیاز به کمک شخصی داشته باشید، می توانید کانال # گیت یا #گیتهاب را در سرور  (irc.freenode.net) IRC نود خالی آزمایش کنید. 
+این کانال ها به طور مرتب با صدها نفر از افرادی که از همه چیز درباره گیت اطلاع دارند و اغلب مایل به کمک هستند پر می شوند.(((IRC)))
 
-In addition, if you don't need the full-blown manpage help, but just need a quick refresher on the available options for a Git command, you can ask for the more concise ``help'' output with the `-h` or `--help` options, as in:
+علاوه بر این، اگر شما نیاز همه جانبه به راهنمای صفحه من نداشته باشید ، بلکه فقط نیازمند یک تازه کننده سریع برای گزینه های موجود برای دستور گیت هستید، می توانید از خروجی `help'' بوسیله دستور" `-h یا help – استفاده کنید، مانند:
 
-[source,console]
+[منبع، کنسول]
 ----
 $ git add -h
 usage: git add [<options>] [--] <pathspec>...
 
     -n, --dry-run         dry run
     -v, --verbose         be verbose
 
-    -i, --interactive     interactive picking
-    -p, --patch           select hunks interactively
-    -e, --edit            edit current diff and apply
-    -f, --force           allow adding otherwise ignored files
-    -u, --update          update tracked files
-    -N, --intent-to-add   record only the fact that the path will be added later
-    -A, --all             add changes from all tracked and untracked files
-    --ignore-removal      ignore paths removed in the working tree (same as --no-all)
-    --refresh             don't add, only refresh the index
-    --ignore-errors       just skip files which cannot be added because of errors
-    --ignore-missing      check if - even missing - files are ignored in dry run
-    --chmod <(+/-)x>      override the executable bit of the listed files
+    -i, --interactive     چیدن تعاملی
```

Comment:
Git core is not translated to farsi. So this is not the output of git.

Reply:
solved